### PR TITLE
Addition of LO3, including cvar to toggle

### PR DESCRIPTION
**Pull request by Technoblazed**
I'm aware this was here before, but you said if someone readded with a cvar you'd readd it, so here you go :)

### CODE REVIEW

**Comment by splewis** [comment]
One of the reasons I didn't want to do this myself is that it could cause issues with other systems: like the stats system (https://github.com/splewis/get5/blob/master/scripting/get5/stats.sp#L52)

I'm not confident that this change wouldn't cause round starts to incorrectly change the roundsplayed stat.

**Comment by Technoblazed** [comment]
I'll be testing it in about 30 minutes, would you like me to get back to you about it?

File: scripting/get5.sp [hunk 1]
```diff
@@ -230,6 +231,8 @@ public void OnPluginStart() {
         "Maximum number of pauses a team can use, 0=unlimited");
     g_MaxPauseTimeCvar = CreateConVar("get5_max_pause_time", "300",
         "Maximum number of time the game can spend paused by a team, 0=unlimited");
+    g_QuickRestartCvar = CreateConVar("get5_quick_restarts", "0",
```
Comment (splewis) [inline scripting/get5.sp]: "quick restarts" isn't a good name for this. get5_use_lo3 would be much descriptive.

File: scripting/get5.sp [hunk 2]
```diff
@@ -231,8 +231,8 @@ public void OnPluginStart() {
         "Maximum number of pauses a team can use, 0=unlimited");
     g_MaxPauseTimeCvar = CreateConVar("get5_max_pause_time", "300",
         "Maximum number of time the game can spend paused by a team, 0=unlimited");
-    g_QuickRestartCvar = CreateConVar("get5_quick_restarts", "0",
-        "Whether to use a quick restart or a full live-on-3 restart. 0 for LO3.");
+    g_QuickRestartCvar = CreateConVar("get5_use_lo3", "0",
+        "Whether to use a quick restart or a full live-on-3 restart.");
```
Comment (splewis) [inline scripting/get5.sp]: I don't think a "quick restart" really describes what happens when this is 0. "Whether to do a live-on-3 restart in addition to the warmup countdown" would be better.

**Comment by Technoblazed** [comment]
Apologies for the delay

**Comment by splewis** [comment]
(don't worry about the merge conflicts I caused)

I think this is still confusing to use, it does the X (default 10) second countdown, then 2 restarts starting at "restart 2/3", which is odd behavior. A fix would be something like also printing "round 1/3" at the initial warmup countdown, though I still don't know if all this is worth the added confusion.

**Comment by splewis** [comment]
Sorry I've left this hanging for so long.

I think I've decided not to add this. The main reason is that I'm trying to avoid including "invasive" things into this plugin. Using the game's builtin warmup system/countdown feels much more natural (even if pepole have used lo3 configs for many years). Another example of an "invasive" thing I don't want would be things like printing all the client money in chat - I'd like to avoid these types of things if there's a good alternative. 

The other reason would the slight awkwardness in the implementation of all the timing of the restarts (arbitrary values and lots of ``CreateTimer``'s). Those are hard to avoid, but I don't think having this option is valuable to deal with all that mess.

**Pull request closed without merging**